pages/about/policies/admissions.html

```diff
@@ -0,0 +1,345 @@
+<p><strong><em>Mount Sion CBS </em></strong></p>
```

Comment:
@evanhearne I think it will look ab it neater if we move the section line1-8 below line 9 as Line 9 is the heading. You can see in the other policies the modal that pops up starts with a heading also.

Reply:
@evanhearne I see that you have removed the lines altogether. I think we should just move it down rather than removing it.

Reply:
@evanhearne was this section added back in?

Reply:
@laurafitzgerald no it was taken out.